Syntaxes/SCSS.sublime-syntax

```diff
@@ -7,379 +7,3624 @@ file_extensions:
+    | wrap(-reverse)?
+    | z-index|zero
+    | zoom(-in|-out)?
+    ){{break}}
+
+  # https://www.w3.org/TR/css-fonts-4/#font-display-desc
+  font_display_constants: |-
+    \b(?xi:
+    block | swap | fallback | optional
+    ){{break}}
+
+  # Generic Font Families
+  font_family_constants: |-
+    \b(?xi:
+    # CSS 2 fonts
+    # https://www.w3.org/TR/CSS22/fonts.html#generic-font-families
+      sans-serif | serif | cursive | monospace | fantasy
+    # CSS 3 level 4 fonts
+    # https://www.w3.org/TR/2019/WD-css-fonts-4-20191113/#generic-family-value
+    | emoji | math | fangsong | system-ui
+    # https://www.w3.org/TR/2019/WD-css-fonts-4-20191113/#standard-font-families
+    | ui-sans-serif | ui-serif | ui-monospace | ui-rounded
+    ){{break}}
+
+  # https://www.w3.org/TR/CSS22/fonts.html#font-shorthand
+  font_property_constants: |-
+    \b(?xi:
+      caption | icon | menu | message-box | small-caption | status-bar
+    ){{break}}
+
+  # https://www.w3.org/TR/CSS22/fonts.html#font-size-props
+  font_size_constants: |-
+    \b(?xi:
+      larger | large | medium | small | smaller | x{1,2}-(?: large | small )
+    ){{break}}
+
+  # https://www.w3.org/TR/CSS22/fonts.html#font-boldness
+  # https://www.w3.org/TR/CSS22/fonts.html#font-styling
+  # https://www.w3.org/TR/CSS22/fonts.html#small-caps
+  font_style_constants: |-
+    \b(?xi:
+      normal | bold | bolder | lighter | italic | oblique | small-caps
+    ){{break}}
+
+###[ PSEUDO SELECTORS FOR SASS ]###############################################
+  pseudo_elements: |-
+    (?x:
+        (:{1,2})(?:before|after|first-line|first-letter) # CSS1 & CSS2 require : or ::
+      | (::)(-(?:moz|ms|webkit)-)?(?:{{ident}}) # CSS3 requires ::
+    ){{break}}
+
+  regular_pseudo_classes: '\b(active|any-link|blank|checked|current|default|defined|disabled|drop|empty|enabled|first|first-child|first-of-type|fullscreen|future|focus|focus-visible|focus-within|host|hover|indeterminate|in-range|invalid|last-child|last-of-type|left|link|local-link|only-child|only-of-type|optional|out-of-range|past|placeholder-shown|read-only|read-write|required|right|root|scope|target|target-within|user-invalid|valid|visited)\b(?![-])'
+  functional_pseudo_classes: '\b(dir|lang|matches|not|has|drop|nth-last-child|nth-child|nth-last-of-type|nth-of-type){{break}}'
+
+
+###############################################################################
 
 contexts:
   main:
-    - include: Sass.sublime-syntax#frontmatter
-    - include: Sass.sublime-syntax#comment-block
-    - include: Sass.sublime-syntax#comment-line
-    - include: Sass.sublime-syntax#selectors
-    - include: properties
-    - include: Sass.sublime-syntax#keyframe-operators
-    - include: Sass.sublime-syntax#interpolated-selectors
-    - include: Sass.sublime-syntax#custom-element-selectors
+    # Yekyll front matter
+    - match: '^---$'
+      scope: frontmatter.jekyll punctuation.section.frontmatter.begin.jekyll
+      embed: scope:source.yaml
```

Comment:
(1) because of this

Reply:
oh, sure, but apparently the 4073 build I was testing against doesn't fail over this but the 4121 build does 🤷🏻